specification/azure-kusto/resource-manager/readme.md

```diff
@@ -133,3 +153,16 @@ java:
 regenerate-manager: true
 generate-interface: true
 ```
+
+### Tag: package-2019-01-21 and java
+
+These settings apply only when `--tag=2019-01-21 --java` is specified on the command line.
+Please also specify `--azure-libraries-for-java=<path to the root directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(tag) == '2019-01-21' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.kusto.v2019-01-21
```

Comment:
use underscore instead of dash (which is invalid char in java namespace).

.v2019-01-21 -> .v2019_01_21